doc/source/users_guide/using-clm-tools/creating-domain-files.rst

```diff
@@ -6,7 +6,7 @@
  Creating CLM domain files
```

Comment:
This can also be removed, but maybe it happens later, TODO

Reply:
TODO added in 6ac23b2.